frontend/src/metabase/public/components/EmbedFrame/LogoBadge/LogoBadge.module.css

```diff
@@ -18,12 +17,15 @@
   color: var(--mb-color-text-primary);
 }
 
+.light {
+  --mb-color-text-primary: var(--mb-color-text-medium);
+}
+
 .dark {
-  --mb-color-text-primary: var(--mb-color-brand-light);
```

Comment:
This color was wrong, we're supposed to just use `text-primary` set by the dark theme.